components/NavigationDrawer/src/MDCBottomDrawerPresentationController.m

```diff
@@ -71,6 +72,10 @@ - (void)presentationTransitionWillBegin {
       [[MDCBottomDrawerContainerViewController alloc]
           initWithOriginalPresentingViewController:self.presentingViewController
                                 trackingScrollView:self.trackingScrollView];
+  if (self.maximumInitialDrawerHeight > 0) {
+    bottomDrawerContainerViewController.maximumInitialDrawerHeight =
+        self.maximumInitialDrawerHeight;
+  }
```

Comment:
Are we able to remove this conditional (keeping the assignment within)? I'm worried about the implication that we can only call this method with non-zero values.